src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drm.c

```diff
@@ -537,6 +542,14 @@ static struct drm_driver mm_drm_driver = {
 	.date				= driver_date,
 };
 
+const struct drm_gem_object_funcs xocl_gem_object_funcs = {
```

Comment:
Please add 5.11 check here as well. I am not sure whether this gets compiled other Kernels or not.